packages/slate-react/src/components/leaf.js

```diff
@@ -142,7 +142,12 @@ class Leaf extends React.Component {
       parent.text === '' &&
       parent.nodes.last() === node
     ) {
-      return <span data-slate-zero-width="n">{'\uFEFF'}</span>
+      return (
+        <span data-slate-zero-width="n">
+          {'\uFEFF'}
+          <br />
```

Comment:
Is there still a need to have _both_ `ZERO WIDTH NO-BREAK SPACE` and `<br />` here? Or could the `<br />` alone be sufficient?

Reply:
That's a good question. I didn't try it with the `<br />` alone in the `<span />`, but replacing the `<span />` with `<br />` does not work